docs/reST/ref/image.rst

```diff
@@ -184,9 +184,22 @@ following formats.
 
    .. ## pygame.image.tostring ##
 
+.. function:: tobytes
+
+   | :sl:`transfer image to byte buffer`
+   | :sg:`tobytes(Surface, format, flipped=False) -> bytes`
+
+   added :func:`tobytes` as an alias for :func:`tostring`,
+   the use of :func:`tobytes` is recommended over :func:`tostring`.
```

Comment:
I'd like this documentation to be a little more helpful, same with the other one 🙂.

What does it do? Why was it added? What should people know?

You've got that it was added, and that it is recommended (what people should know).